src/MakingSense.AspNetCore.Documentation/DocumentationOptions.cs

```diff
@@ -27,11 +27,13 @@ public class DocumentationOptions
 		public IFileProvider FileProvider { get; set; }
 
 		/// <summary>
-		/// Not found handling is disabled by default, set NotFoundHtmlFile to enable it.
+		/// Not found handling is disabled by default, set NotFoundHtmlPath to enable it.
 		/// </summary>
 		public bool EnableNotFoundHandling => NotFoundHtmlFile != null && NotFoundHtmlFile.Exists;
 
-		public IFileInfo NotFoundHtmlFile { get; set; }
+		public IFileInfo NotFoundHtmlFile { get; internal set; }
```

Comment:
This introduces a breaking change, do you think that it is possible to mark set of `NotFoundHtmlFile` and `LayoutFile` properties obsolete keep old behavior still working?

Reply:
You are right, I'll take a look